homeassistant/components/dlna_dms/manifest.json

```diff
@@ -4,7 +4,8 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/dlna_dms",
   "requirements": ["async-upnp-client==0.23.5"],
-  "dependencies": ["media_source", "ssdp"],
+  "dependencies": ["ssdp"],
+  "after_dependencies": ["media_source"],
```

Comment:
This is odd... as the sole goal of dlna_dms is media source...

Reply:
Radio Browser also doesn't have a `media_source` dependency. 

Just like we don't have a `websocket_api` dependency.

We should be able to assume that people have `default_config` or added `media_source` back.